test/src/test/java/jenkins/install/LoadDetachedPluginsTest.java

```diff
@@ -87,62 +86,6 @@ public void upgradeFromJenkins2() {
         });
     }
 
-    @Issue("JENKINS-48604")
-    @Test
-    @LocalData
-    public void upgradeFromJenkins2WithNewerDependency() {
```

Comment:
@dwnusbaum FYI I think these were your tests. Not sure how to rework them to be less brittle.

Reply:
Yeah, I don't have any good idea myself. The tests either need to use actual detached plugins or dynamically set up some simple, self-contained plugins as if they were detached plugins and use those in tests. The first option is brittle, and the second seems complicated.

Reply:
Right, the second option sort of presumes a test harness that would let you work with a specially-built WAR file. Beyond the power of `JenkinsRule`.